deploy/helm/sumologic/conf/setup/custom.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# This script copies files from /customer-scripts to /scripts/<dirname> basing on the filename
+#
+# Example file structure:
+#
+# /customer-scripts
+# ├── dir1_main.tf
+# ├── dir1_setup.sh
+# ├── dir2_list.txt
+# └── dir2_setup.sh
+#
+# Expected structure:
+#
+# /scripts
+# ├── dir1
+# │   ├── main.tf
+# │   └── setup.sh
+# └── dir2
+#     ├── list.txt
+#     └── setup.sh
+#
+# shellcheck disable=SC2010
+# extract target directory names from the file names using _ as separator
+for dir in $(ls -1 /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g' | sort | uniq); do
+  target="/scripts/${dir}"
+  mkdir "${target}"
+  # shellcheck disable=SC2010
+  # Get files for given directory and take only filename part (after first _)
+  for file in $(ls -1 "/customer-scripts/${dir}_"* | grep -oE '_.*' | sed 's/_//g'); do
+    cp "/customer-scripts/${dir}_${file}" "${target}/${file}"
+  done
+  cd "${target}" && bash setup.sh
```

Comment:
Ok so this will work but it will keep the changed CWD from the last `dir` in the loop hence the directory for the caller will be changed.

To fix that we can e.g. save the dir before the loop and then `popd` or `cd` to it at the end

Reply:
I'm operating on absolute paths, so `CWD` inside the `custom.sh` doesn't impact on anything